Resources/app/Administration/test/e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,130 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add("login", (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This is will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+/**
+ * Switches administration UI locale to EN_GB
+ * @memberOf Cypress.Chainable#
+ * @name setLocaleToEnGb
+ * @function
+ */
+Cypress.Commands.add('setLocaleToEnGb', () => {
+    return cy.window().then((win) => {
+        win.localStorage.setItem('sw-admin-locale', Cypress.env('locale'));
+    });
+});
+
+/**
+ * Cleans up any previous state by restoring database and clearing caches
+ * @memberOf Cypress.Chainable#
+ * @name openInitialPage
+ * @function
+ */
+Cypress.Commands.add('openInitialPage', (url) => {
```

Comment:
don't we already have commands in the base package for this things? they look quite common